### ISSUE TITLE
Feature Request: compact mode media flyout

### ISSUE THREAD

**Issue by Samuel12321** [ShankarBUS/ModernFlyouts#148]
Reintroduce the 0.2 media flyout (modified to match current style) as a compact mode flyout. 
![Inked97527904-4fc4dd00-1a11-11eb-9ec6-5f027119a523_LI](https://user-images.githubusercontent.com/35312698/97528163-ea252080-1a11-11eb-9218-7d5c481ef260.jpg)
Made more compact by removing what i drew over.

Could be useful for people who either have a lot of different media sources open or people who just want a smaller flyout.

**Comment by ShankarBUS** [comment]
I don't think this will be necessary and be an easy implementation.
Let's get some community feedback first.

**Comment by Cyberdroid1** [comment]
Yeah, well I partly support your point, Sam. Have that minimal size would be great. But since Shankar said, it won't be that easy.

**Comment by ShankarBUS** [comment]
@Cyberdroid1,

I'm halfway done with #49.
After looking at the design, it seems kinda possible.

I just need some time to plan this out. Not for v0.8 or v0.9 though 🤷‍♂️. This may take a bit longer.

**Comment by Samuel12321** [comment]
definitely low priority.

**Comment by Cyberdroid1** [comment]
@ShankarBUS You can add this to your box of surprises for v 0.10 😁

**Comment by Samuel12321** [comment]
tentatively added this to the 0.10 release. But we can move it up or down.

**Comment by ShankarBUS** [comment]
@Cyberdroid1 oooooh nice!

I completely forgot I said that I'm gonna do some suprise features in v0.10.0! 😂
